src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(digit) {
+      return (this.result += digit);
+    },
+    subtract(digit) {
+      return (this.result -= digit);
```

Comment:
Similar to the `add` method, the `subtract` method should not return the result for the same reason.